README.md

```diff
@@ -831,6 +831,23 @@ mrsk lock acquire -m "Doing maintanence"
 mrsk lock release
 ```
 
+## Gradual restarts
+
+When deploying to large numbers of hosts, you might prefer not to restart your services on every host at the same time.
```

Comment:
That's true. I'd included `start` because bringing up a set of new hosts could have a similar effect. But that's probably rare, and if they're actually new hosts you could also just add them in batches.

Will change to boot only 👍

Reply:
Changed in https://github.com/mrsked/mrsk/pull/222/commits/100b72e4b48d4a887746ddf0f8981b1de1096dcf